### DISCUSSION TITLE
Previous output is also getting showed even after running the code again in notebook in vscode

### DISCUSSION THREAD

**Discussion by Ransinghsatyajitray** [microsoft/vscode-jupyter#15070] [Questions and Answers]
When I am running a line of code inside notebook in vscode, even the previous output is also getting shown. Any help with respect to how to ensure that when the code is run only the lastest output is reflected and not any output from the previous run of the cell
![Issue](https://github.com/microsoft/vscode-jupyter/assets/30564953/11bee28a-cb7c-4b09-b91a-f3b3f3577427)

**Comment by amunger** [comment]
I believe this was an issue with a specific 3rd party renderer.
